zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -236,7 +236,254 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-
+  /users:
```

Comment:
🔧 Please provide proper `description` fields for all added objects, e.g. endpoints, schemas, and object attributes. We'll use this file to generate the public API explorer in https://github.com/camunda/camunda-docs so any missing information will lead to a bad public documentation experience.

Reply:
Feel free to create a follow-up issue for this, if you're otherwise blocked. Merging this PR in general is more important than this specific point. We need to tackle it, but it's not essential for this PR.

Reply:
Makes sense, done that:
https://github.com/camunda-cloud/identity/issues/2978